backend/src/yeomanui.ts

```diff
@@ -116,6 +116,24 @@ export class YeomanUI {
 		await this.rpc.invoke("updateGeneratorsPrompt", [generators.questions]);
 	}
 
+	public async _notifyGeneratorsInstall(args: any[], force: boolean) {
+		if (!_.isNil(args)) {
+			const isGeneratorsPrompt: boolean = await this.rpc.invoke("isGeneratorsPrompt");
+			if (isGeneratorsPrompt || force) {
+				const message = this.getGeneratorsInstallingMessage(args);
+				this.youiEvents.getAppWizard().showInformation(message, MessageType.prompt);
+			}
+		}
+		this.uiOptions.installGens = (_.isObject(args) && _.isEmpty(args)) ? undefined : args;
+	}
+
+	private getGeneratorsInstallingMessage(args: any[]): string {
+		if (_.isEmpty(args)) {
+			return "We have finished installing all generators, enjoy your work!";
```

Comment:
Is it worth to extract this message to messages file ?

Reply:
Yes. after Paola will review. + in a new PR I'll try notifying which gens are being installed.